### PR TITLE
https://issues.redhat.com/browse/ACM-6084 Clarify policy namespaces

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.7.

### CODE REVIEW

**Comment by ch-stark** [comment]
looks fine for me thanks @oafischer

**Review by mprahl — APPROVED** [review]

File: governance/policy_overview.adoc [hunk 1]
```diff
@@ -9,11 +9,10 @@ The policy requires a _PlacementRule_ or _Placement_ that defines the clusters t
 
 *Important:*
 * You must create the `PlacementBinding` to bind your policy with either a `PlacementRule` (deprecated) or a `Placement` in order to propagate the policy to the managed clusters.
-+
+
 *Best practice:* Use the command line interface (CLI) to make updates to the policies when you use the `Placement` resource.
 
-* You can create a policy in any namespace on the hub cluster except the cluster namespace.
-If you create a policy in the cluster namespace, it is deleted by {product-title}.
+* You can create a policy in any namespace on the hub cluster except the cluster namespace. Every managed cluster already has a namespace on the hub cluster. If you create a policy in the cluster namespace, it is deleted by {product-title}. Instead, use a namespace such as `policy`.
```
Comment (cadawson) [inline governance/policy_overview.adoc]: - "...except the cluster namespace." Do we need to distinguish hub or managed here? Any cluster namespace? 
- "Every managed cluster has a namespace on the hub cluster." Suggestion to consider removing _already_.

**Review by cadawson — APPROVED** [review]
A couple of considerations. Thanks!

File: governance/policy_overview.adoc (hunk 1 above)
Comment (oafischer) [reply governance/policy_overview.adoc]: @ch-stark Do you think it is still clear enough after translation if we remove "already"?

Comment (oafischer) [reply governance/policy_overview.adoc]: Probably better to specify managed here, I'll update it. Thanks!

Comment (dhaiducek) [reply governance/policy_overview.adoc]: I agree with @cadawson. I think "already" can be more specific. Maybe something like this?

>* You can create a policy in any namespace on the hub cluster except the managed cluster namespaces. Every managed cluster has a namespace on the hub cluster matching the managed cluster's name. If you create a policy in a managed cluster namespace, it is deleted by {product-title}. Instead, use a namespace such as `policy`.

Comment (oafischer) [reply governance/policy_overview.adoc]: Thanks, I'll update based on the suggestion!

File: governance/policy_overview.adoc [hunk 2]
```diff
@@ -9,11 +9,10 @@ The policy requires a _PlacementRule_ or _Placement_ that defines the clusters t
 
 *Important:*
 * You must create the `PlacementBinding` to bind your policy with either a `PlacementRule` (deprecated) or a `Placement` in order to propagate the policy to the managed clusters.
-+
+
 *Best practice:* Use the command line interface (CLI) to make updates to the policies when you use the `Placement` resource.
 
-* You can create a policy in any namespace on the hub cluster except the cluster namespace.
-If you create a policy in the cluster namespace, it is deleted by {product-title}.
+* You can create a policy in any namespace on the hub cluster except the managed cluster namespace. Every managed cluster has a namespace on the hub cluster that matches the managed cluster name. If you create a policy in the managed cluster namespace, it is deleted by {product-title}. Instead, use a namespace such as `policy`.
```
Comment (dhaiducek) [inline governance/policy_overview.adoc]: Minor wording suggestion
```suggestion
* You can create a policy in any namespace on the hub cluster except the managed cluster namespaces. Every managed cluster has a namespace on the hub cluster that matches the managed cluster's name. If you create a policy in a managed cluster namespace, it is deleted by {product-title}. Instead, use a namespace such as `policy`.
```

**Review by dhaiducek — APPROVED** [review]

File: governance/policy_overview.adoc [hunk 3]
```diff
@@ -9,11 +9,10 @@ The policy requires a _PlacementRule_ or _Placement_ that defines the clusters t
 
 *Important:*
 * You must create the `PlacementBinding` to bind your policy with either a `PlacementRule` (deprecated) or a `Placement` in order to propagate the policy to the managed clusters.
-+
+
 *Best practice:* Use the command line interface (CLI) to make updates to the policies when you use the `Placement` resource.
 
-* You can create a policy in any namespace on the hub cluster except the cluster namespace.
-If you create a policy in the cluster namespace, it is deleted by {product-title}.
+* You can create a policy in any namespace on the hub cluster except the managed cluster namespaces. Every managed cluster has a namespace on the hub cluster that matches the managed cluster's name. If you create a policy in a managed cluster namespace, it is deleted by {product-title}. Instead, use a namespace such as `policy`.
```
Comment (dockerymick) [inline governance/policy_overview.adoc]:
```suggestion
* You can create a policy in any namespace on the hub cluster except the managed cluster namespaces. Every managed cluster has a namespace on the hub cluster that matches the name of the managed cluster. If you create a policy in a managed cluster namespace, it is deleted by {product-title}. Instead, use a namespace such as `policy`.
```

**Pull request merged**